Packs/JOB-GetAtRiskIncidents/Scripts/AtRiskIncidentsNotification/AtRiskIncidentsNotification.yml

```
@@ -0,0 +1,28 @@
+commonfields:
+  id: fbeb69d7-1142-4f06-8e2e-3504d6c84695
```

```suggestion
  id: At-Risk-Incidents-Notification
```
the id should match the name.
not an obligation: the name of a script/integration/etc better not contain characters like "-", so would be better if the name is "At Risk Incidents Notification" (the id should match it as well)